modules/core/arrow-typeclasses/src/main/kotlin/arrow/typeclasses/suspended/prelude.kt

```diff
@@ -0,0 +1,38 @@
+package arrow.typeclasses.suspended
+
+import arrow.Kind
+
+interface Prelude {
```

Comment:
so it's aggregated and the function is defined in the restricted scope of the continuation otherwise `effectIdentity` would need to also become lifted each time into the monad context.